### ISSUE TITLE
Daily builds not updating

### ISSUE THREAD

**Issue by MauricioFauth** [phpmyadmin/website#178]
Both phpMyAdmin 6.0+snapshot and phpMyAdmin 5.2+snapshot are not updating.

```
./monitoring/check_phpmyadmin_daily_snapshots.py
Problem found
Current date: 2024-07-18
Snapshot date: 2024-05-23
Problem found
Current date: 2024-07-18
Snapshot date: 2024-05-23
```

**Comment by liviuconcioiu** [comment]
There was a previous issue until last week, but I don't see any code change that fixed it?!  - https://github.com/phpmyadmin/phpmyadmin/issues/19183.

Right now there's another issue - https://github.com/phpmyadmin/phpmyadmin/actions/runs/9983222678/job/27590417830#step:12:3606

**Comment by MauricioFauth** [comment]
> Right now there's another issue - https://github.com/phpmyadmin/phpmyadmin/actions/runs/9983222678/job/27590417830#step:12:3606

I'm not sure if the Check release excludes script causes the build to fail.

**Comment by williamdes** [comment]
Daily snapshots do not run it
Please send me a ssh public key on matrix so I can give you server access

**Comment by williamdes** [comment]
We should fix the snaphosts and releases script for 6.0 as they seem to include dev vendors like my script reported back to us

**Comment by MauricioFauth** [comment]
Fixed
